### PR TITLE
Enable jit by default on PPC64

### PR DESCRIPTION
**Pull request by brunoalr**

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: hphp/runtime/base/runtime-option.cpp [hunk 1]
```diff
@@ -452,7 +452,7 @@ static inline uint64_t pgoThresholdDefault() {
 
 static inline bool evalJitDefault() {
 // Disable JIT for PPC64 - Port under development
```
Comment (WizKid) [inline hphp/runtime/base/runtime-option.cpp]: What about this comment?

Comment (brunoalr) [inline hphp/runtime/base/runtime-option.cpp]: Removed the comment. Jit is already working on PPC64,

**Comment by fredemmott** [comment]
@facebook-github-bot import

**Comment by fredemmott** [comment]
@facebook-github-bot import

**Comment by fredemmott** [comment]
@facebook-github-bot import

**Comment by brunoalr** [comment]
@facebook-github-bot import

**Comment by brunoalr** [comment]
@WizKid I've been recently added to my company's corporate CLA and talked to @Orvid in #hhvm-dev about importing this pull request to Phabricator. Can you check the current status, please?

**Comment by WizKid** [comment]
@brunoalr I don't work on HHVM. I just read the some diffs because I think it is interesting. @Orvid knows what to do

**Comment by brunoalr** [comment]
@WizKid no problem! thanks for pointing that out

**Comment by facebook-github-bot-2** [comment]
This pull request has been imported into Phabricator, and discussion and review of the diff will take place at https://reviews.facebook.net/D61881

**Comment by facebook-github-bot-9** [comment]
This pull request has been imported into Phabricator, and discussion and review of the diff will take place at https://reviews.facebook.net/D61887

**Comment by facebook-github-bot-2** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by facebook-github-bot-9** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Pull request closed without merging**